src/mobiflight.cpp

```diff
@@ -9,6 +9,7 @@
 #include "Button.h"
 #include "Encoder.h"
 #include "MFEEPROM.h"
+#include "ArduinoUniqueID.h"
```

Comment:
I don't think this needs to be included in mobiflight.cpp since `UniqueId` isn't used anywhere in the file.

Reply:
Oh yes, that's an fragment from my testing.